store/sqlstore/system_store.go

```diff
@@ -98,7 +99,9 @@ func (s SqlSystemStore) Get() (model.StringMap, error) {
 
 func (s SqlSystemStore) GetByName(name string) (*model.System, error) {
 	var system model.System
-	if err := s.GetMaster().SelectOne(&system, "SELECT * FROM Systems WHERE Name = :Name", map[string]interface{}{"Name": name}); err != nil {
+	if err := s.GetMaster().SelectOne(&system, "SELECT * FROM Systems WHERE Name = :Name", map[string]interface{}{"Name": name}); err == sql.ErrNoRows {
+		return nil, store.NewErrNotFound("System", fmt.Sprintf("name=%s", system.Name))
+	} else if err != nil {
```

Comment:
Should we stick with the standard pattern that is used in the code base here?

```suggestion
	if err := s.GetMaster().SelectOne(&system, "SELECT * FROM Systems WHERE Name = :Name", map[string]interface{}{"Name": name}); err != nil {
	if err == sql.ErrNoRows {
		return nil, store.NewErrNotFound("System", fmt.Sprintf("name=%s", system.Name))
	}
```

Reply:
If we were going forward with this change, I would be more comfortable with a Server Platform team member review.

Reply:
Adding @streamer45 from the Server team to the review.